docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -672,6 +672,15 @@ The following deprecated Twig classes have been replaced:
 |`Twig_Template`|`Twig\Template`|
 |`Twig_Node`|`Twig\Node\Node`|
 
+### Password hashes
+
+Insecure password hash types deprecated since v1.13 have been removed:
```

Comment:
```suggestion
Insecure password hash types deprecated since v1.13 have been removed. Login with such hashes will not work. Users can request a new, valid password using the "Forgot password" feature. The removed types are:
```